package.json

```diff
@@ -30,5 +30,17 @@
     "tsc-alias": "^1.7.0",
     "tsconfig-paths": "^4.1.0"
   },
-  "dependencies": {}
+  "dependencies": {
```

Comment:
I think we don't need sdk deps here, since those are configured in packages/sdk/package.json